values/k8s/k8s-raw.gotmpl

```diff
@@ -74,6 +74,7 @@ resources:
     - kind: ServiceAccount
       name: default
       namespace: team-admin
+  {{- if eq $v.otomi.mode "ee" }}
```

Comment:
I am trying to keep that flag out, as we should just check for the existence of the pullsecret. It was not quickly possible in our setup to nullify it

Reply:
I do not understand your concept. What would you like to achieve? And does it gonna help other users/developers?

Reply:
don't use that flag, it will be removed

Reply:
ok, I will change the code but next time please mark this field as deprecated and communicate it to the team.

Reply:
done